### DISCUSSION TITLE
#bug mubu doesn't like polybuffer~

### DISCUSSION THREAD

**Discussion by mattyostrowski** [Ircam-RnD/catart-mubu#110] [Bug, Issues, and Suggestions]
Hi guys,

Have a look at the (poly)buffer~ tab of the MuBu help file, and do the following:
-- fill the polybuffer~
-- press 'refer help-mubu-pobu' message
Result: You'll see anton.aif in the imubu.
-- select a different buffer, either via the bufferindex message or in the imubu itself
Result: Blank imubu, not even a tab
-- go back to buffer 1
Result: buffer 1 now blank.

I don't think it's just a display thing, as I can't seem to hear anything using mubu.play~

Just letting you know!

\M

**Comment by christophertrapani** [comment]
I'm finding that I can't replicate this issue: I'm able to select different
buffers, and to return to buffer 1, which isn't blank.

**Comment by mattyostrowski** [comment]
Okay, I see. I had some kind of hotfix mubu.mxo, which I dimly remember getting when I was there in October for some reason. I just reinstalled from the package manager, and the version I was having trouble with has a creation date a couple of days earlier than the current release version, so that’s it. Sorry for the noise.

\M




On December 14, 2023 at 18:03:01, Christopher Trapani ***@***.***) wrote:

I'm finding that I can't replicate this issue: I'm able to select different  
buffers, and to return to buffer 1, which isn't blank.

**Comment by diemoschwarz** [comment]
Thanks for checking, Chris!  Matt, could you please mark this as solved also on 
the Forum?

On 15/12/2023 01:28, mattyostrowski wrote:
> Okay, I see. I had some kind of hotfix mubu.mxo, which I dimly remember getting 
> when I was there in October for some reason. I just reinstalled from the package 
> manager, and the version I was having trouble with has a creation date a couple 
> of days earlier than the current release version, so that’s it. Sorry for the 
> noise.
> 
> \M
> 
> 
> 
> 
> On December 14, 2023 at 18:03:01, Christopher Trapani ***@***.***) wrote:
> 
> I'm finding that I can't replicate this issue: I'm able to select different
> buffers, and to return to buffer 1, which isn't blank.
> 
> On Thu, Dec 14, 2023 at 4:37 PM mattyostrowski ***@***.***>
> wrote:
> 
>  > Hi guys,
>  >
>  > Have a look at the (poly)buffer~ tab of the MuBu help file, and do the
>  > following:
>  > -- fill the polybuffer~
>  > -- press 'refer help-mubu-pobu' message
>  > Result: You'll see anton.aif in the imubu.
>  > -- select a different buffer, either via the bufferindex message or in the
>  > imubu itself
>  > Result: Blank imubu, not even a tab
>  > -- go back to buffer 1
>  > Result: buffer 1 now blank.
>  >
>  > I don't think it's just a display thing, as I can't seem to hear anything
>  > using mubu.play~
>  >
>  > Just letting you know!
>  >
>  > \M
>  >
>  > —
>  > Reply to this email directly, view it on GitHub
>  > <https://github.com/ircam-ismm/catart-mubu/discussions/110>, or
>  > unsubscribe
>  > 
> <https://github.com/notifications/unsubscribe-auth/AIRYDC7LFROKRX2EBRZGDF3YJN5RBAVCNFSM6AAAAABAVSHPVCVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZVHE3DCOJZHE> 
> 
>  > .
>  > You are receiving this because you are subscribed to this thread.Message
>  > ID: ***@***.***>
>  >
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you authored the thread.
> 
> —
> Reply to this email directly, view it on GitHub 
> <https://github.com/ircam-ismm/catart-mubu/discussions/110#discussioncomment-7859026>, 
> or unsubscribe 
> <https://github.com/notifications/unsubscribe-auth/AAC4KJ2WPYP4SIAZWQKKJ7TYJOKUHAVCNFSM6AAAAABAVSHPVCVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TQNJZGAZDM>.
> You are receiving this because you are subscribed to this thread.Message ID: 
> ***@***.***>
> 

...
								...Diemo

-- 
Diemo Schwarz, PhD -- http://diemo.concatenative.net
https://www.imera.fr/chercheur/schwarz-diemo/
https://www.stms-lab.fr/person/diemo-schwarz
Sound–Music–Movement Interaction Team -- http://ismm.ircam.fr
IRCAM - Centre Pompidou -- 1, place Igor-Stravinsky, 75004 Paris, France
Phone +33-1-4478-4879 -- Fax +33-1-4478-1540
